api/v1beta1/ssp_webhook.go

```diff
@@ -91,3 +105,71 @@ func (r *SSP) ValidateDelete() error {
 func setClientForWebhook(c client.Client) {
 	clt = c
 }
+
+func validatePlacement(ssp *SSP) error {
+	err := validateOperandPlacement(ssp.Spec.NodeLabeller.Placement)
+	if err != nil {
+		return err
+	}
+
+	err = validateOperandPlacement(ssp.Spec.TemplateValidator.Placement)
+	if err != nil {
+		return err
+	}
+
+	return nil
+}
+
+func validateOperandPlacement(placement *api.NodePlacement) error {
+	if placement == nil {
```

Comment:
Shouldn't this return an error saying that there is not placement rules to be applied?

Reply:
No. No placement rules is acceptable, it basically tells the k8s scheduler that we don't care where our workloads are placed, we just want them running